server/service/osquery_utils/queries.go

```diff
@@ -150,54 +150,13 @@ limit 1
 	"os_version_windows": {
 		// Windows-specific registry query is required to populate `host.OSVersion` for Windows.
 		Query: `
-WITH dv AS (
 	SELECT
-		data
-	FROM
-		registry
-	WHERE
-		path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\DisplayVersion'
-),
-rid AS (
-	SELECT
-		data
+		os.name,
```

Comment:
Wow I'm surprised the Windows queries can be simplified that much! That's great! I can't judge the new queries vs old, I'm assuming we have good reason for that change and have tested the new ones work as expected.

Reply:
This change is by the hand of @marcosd4h , things don't get any more windows expert than that :)

Reply:
Whoa! So much cleaner ✨